python/mxnet/amp/lists/symbol.py

```diff
@@ -0,0 +1,175 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# coding: utf-8
+"""Lists of functions whitelisted/blacklisted for automatic mixed precision in symbol API."""
+
+FP16_FUNCS = [
```

Comment:
There are operators neither whitelisted nor blacklisted. Can users expect those operators to work with AMP? If a new operator is contributed to MXNet from the community, can we have some kind of general guideline (either documentation here or external resources) on when an operator should be added to one of these lists?

Reply:
Operators that are neither white- nor blacklisted are considered to be neutral to dtype and so they are just handled by normal type inference in the framework (if the input was fp16 then they will be in fp16, if the input was fp32 then it will be fp32).

Reply:
New fp32 operators are introduced into MXNet frequently, and the author may not know `FP32_FUNCS` exists and didn't add it into this list, then the new added operators would be neither white- nor blacklisted. When they're used with AMP, this behavior may try to run it in low precision mode, causing unexpected result. So maybe we can create `FP16_FUNCS` and `FP16_FP32_FUNCS` here, making non-listed operators are FP32 only.